collection/endpoint.csv

```diff
@@ -1194,6 +1194,59 @@ fbec7b1b621bbb7d9152aa1a52028dd914d8bb78b0fa38683722b30a6c115330,https://www.tor
 cfbbcf314651e2199d603e5301aeaa0c4259f63a7aaa8fdb4990ad17dc7964a5,https://www.wychavon.gov.uk/component/fileman/file/Documents/Planning/Brownfield%20Land%20Register%20Sites/WDC%20BLR%202022%20Accessible%20table%20excel.xlsx?routed=1,,,2023-08-10T14:14:10:00Z,,
 de69ac62e9607c01d12d73e13bcf8461f514d59de3de0fb270e7dc4fd20501b9,https://www.westminster.gov.uk/media/document/city-of-westminsters-brownfield-register-2023,,,2023-08-10T14:14:21:28Z,,
 571e11229ab26af7a0467b81d386a8d555a584baa4c3d4a867ac031ee3d0feb9,https://www.westsuffolk.gov.uk/planning/Planning_Policies/upload/Brownfield-Register-2022-final.csv,,,2023-08-10T14:14:32:32Z,,
-5740e23abaf75903d32e534274702bee8b09481b0ed3973c0fe91d88b39201bd,https://www.warrington.gov.uk/sites/default/files/2023-04/Brownfield%20Land%20Register%202022%20-%20CSV.csv,,,2023-08-10T14:14:39:09Z,,

```

Comment:
If you are wondering why this was removed, it's because the lookup originally didn't have the local authority in it and I added it myself